python/pyspark/sql/session.py

```diff
@@ -182,6 +185,7 @@ def getOrCreate(self):
                     session.sparkContext._conf.set(key, value)
                 return session
 
+    #: class variable for :class:`Builder` shared by all :class:`SparkSession` instances
```

Comment:
I think we should rather resemble:

https://github.com/apache/spark/blob/d6ee69e7761a62e47e21e4c2ce1bb20038d745b6/sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala#L970

and I'd use the term "class attribute" instead of "class variable"`

Reply:
Yep.